### ISSUE TITLE
Exclude some lines from being parsed as comments (i.e., shebangs)

### ISSUE THREAD

**Issue by ckald** [ckald/pyccoon#7]
Right now, meta comments are parsed as documentation, which is okay, but it might be nicer to exclude them from docs:

```
# -- coding: utf-8 --
#!/usr/bin/python
```

InlineCommentLanguage should implement this feature and inheriting objects would specify the exact regexps to ignore

**Comment by cryptonomicon314** [comment]
I agree with this. Allowing pyccoon to ignore certain regexps seems perfect. You might want to add these capabilities to MultilineCommentLanguage too, to handle things like Haskell's LANGUAGE (https://downloads.haskell.org/~ghc/7.2.2/docs/html/users_guide/pragmas.html) pragmas, which are comments, but are parsed by the compiler and certainly not documentation.

**Comment by cryptonomicon314** [comment]
Fixed in my pull request (https://github.com/ckald/pyccoon/pull/5) for inline comments, but not for multiline comments yet.
Only special python comments are supported, but it is easy to add support for other languages.
